### ISSUE TITLE
Deleting API-keys and labels should require confirmation

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#953]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
Deleting API-keys and labels should be harder than it is today

### Description
Deleting API-keys and labels can have serious consequences (for integrations, etc.)
Right now, all that's required is clicking "Delete".
See https://didapp-dev.azurewebsites.net/admin/apitokens
and https://didapp-dev.azurewebsites.net/admin/labels

![deletekey](https://user-images.githubusercontent.com/7300548/116084270-e7904480-a69d-11eb-9a73-426aadf4020a.gif)

Deleting should require at least two steps, i.e. a confirmation dialog ("Are you sure you want to delete this [API-key|label]?")

### Proposed solution
Implement a simple confirmation dialog before deleting the API-key
